builtin/providers/aws/resource_aws_efs_mount_target.go

```diff
@@ -161,22 +161,30 @@ func resourceAwsEfsMountTargetRead(d *schema.ResourceData, meta interface{}) err
 		MountTargetId: aws.String(d.Id()),
 	})
 	if err != nil {
-		return err
+		if ec2err, ok := err.(awserr.Error); ok && ec2err.Code() == "MountTargetNotFound" {
```

Comment:
I'd like to see a test for _disappears - look at how db_instance tests work for this if you fancy trying it :)